firefox_ui_tests/functional/toolbar/manifest.ini

```diff
@@ -1,2 +1,3 @@
 [test_back_forward.py]
 [test_home_button.py]
+[test_access_location_bar.py]
```

Comment:
We want to put those tests under functional/locationbar. Please use `git mv` to get the test moved over, and the manifest updated. Please also ensure you sort the tests alphabetical.